app/views/projects/blob/_replace.html.haml

```diff
@@ -0,0 +1,24 @@
+#modal-replace-blob.modal
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %a.close{href: "#", "data-dismiss" => "modal"} ×
+        %h3.page-title Replace #{@blob.name}
+        %p.light
+          From branch
+          %strong= @ref
+      .modal-body
+        = form_tag namespace_project_blob_path(@project.namespace, @project, @id), method: :put, class: 'form-horizontal', :multipart => true  do
+          %br
+          .form-group
+            .col-sm-offset.2.col-sm-10
```

Comment:
This line is wrong. It should be `.col-sm-offset-2.col-sm-10` (note the dash before the 2).